testsuite/MDAnalysisTests/analysis/test_density.py

```diff
@@ -36,8 +36,8 @@
 import MDAnalysis as mda
 from MDAnalysis.analysis import density
 
-from MDAnalysisTests.datafiles import TPR, XTC, GRO
-from mock import Mock, patch
+from MDAnalysisTests.datafiles import TPR, XTC, GRO, PDB_full
+from unittest.mock import Mock, patch
```

Comment:
Another one I failed to see in @wvandertoorn's PR. This import seems to be making Travis' python 2.7 tests fail. If you can add it in a new PR @wvandertoorn that would be great.